headers/functions/arm7.h

```diff
@@ -2,11 +2,15 @@
 #define HEADERS_FUNCTIONS_ARM7_H_
 
 void EntryArm7(void);
+void MainArm7(void);
 int ClearIrqFlag(void);
 int EnableIrqFlag(void);
 int SetIrqFlag(int new_value);
 int EnableIrqFiqFlags(void);
 int SetIrqFiqFlags(int new_value);
 int GetProcessorMode(void);
+unsigned long long __divsi3(int dividend, int divisor);
```

Comment:
Hmm okay I don't think this copy-pasting from arm9 to arm7 is going to be sustainable. Can we delete the duplicate function prototypes from `arm7.h` and change the descriptions in `arm7.yml` to say something like:

```
Copy of the ARM9 function. See arm9.yml for more information.
```
?

Reply:
Sure